### ISSUE TITLE
Some parameters are missing type descriptions

### ISSUE THREAD

**Issue by cheyennee** [tensorflow/tensorflow#60921]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Documentation Bug

### Have you reproduced the bug with TF nightly?

Yes

### Source

source

### Tensorflow Version

tf 2.12.0

### Custom Code

Yes

### Current Behaviour?

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/pigpi/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/pigpi/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

API | lack of type desciption params
-- | --
tf.sparse.bincount | values
tf.keras.layers.Input | tensor
tf.keras.applications.DenseNet121 | input_tensor
tf.math.add_n | inputs
tf.keras.applications.MobileNetV2 | input_tensor
tf.device | device_name
tf.keras.regularizers.get | identifier
tf.metrics.RootMeanSquaredError | metrics
tf.expand_dims | input
tf.keras.applications.DenseNet201 | input_tensor
tf.signal.frame | signal
tf.losses.mean_squared_error | y_true 、 y_pred
tf.losses.cosine_similarity | y_true 、 y_pred
tf.keras.metrics.binary_accuracy | y_true 、 y_pred
tf.keras.losses.logcosh | y_true 、 y_pred
tf.keras.applications.EfficientNetB5 | input_tensor
tf.metrics.binary_accuracy | y_true 、 y_pred
tf.data.experimental.assert_cardinality | expected_cardinality
tf.keras.metrics.sparse_top_k_categorical_accuracy | y_true 、 y_pred
tf.losses.squared_hinge | y_true 、 y_pred
tf.keras.utils.pack_x_y_sample_weight | x、y、sample_weight
tf.keras.activations.softplus | x
tf.nn.depth_to_space | input
tf.gather_nd | params
tf.zeros_like | input
tf.keras.applications.EfficientNetB2 | input_tensor
tf.keras.metrics.MeanAbsoluteError | metrics
tf.stack | values
tf.ensure_shape | x
tf.roll | input
tf.keras.activations.gelu | x
tf.linalg.set_diag | input、diagonal、k
tf.math.bincount | arr、weights
tf.concat | values
tf.keras.applications.EfficientNetB6 | input_tensor
tf.keras.losses.mean_squared_error | y_true 、 y_pred
tf.random.categorical | logits
tf.keras.backend.is_keras_tensor | x
tf.keras.activations.tanh | x
tf.pad | tensor
tf.keras.initializers.identity | gain
tf.keras.applications.EfficientNetB0 | input_tensor
tf.repeat | input
tf.image.convert_image_dtype | image
tf.split | value
tf.keras.utils.to_categorical | y
tf.scatter_nd | updates
tf.keras.layers.concatenate | input
tf.linalg.banded_triangular_solve | bands、rhs
tf.ones_like | input
tf.nest.flatten | structure
tf.keras.activations.linear | x
tf.linalg.tensor_diag_part | input
tf.image.pad_to_bounding_box | image
tf.config.set_visible_devices | devices
tf.size | input
tf.image.resize | images
tf.tile | input
tf.keras.applications.VGG16 | input_tensor
tf.keras.metrics.top_k_categorical_accuracy | y_true 、 y_pred
tf.initializers.identity | gain
tf.image.stateless_random_brightness | image
tf.ragged.range | starts、limits、deltas
tf.reshape | tensor
tf.losses.logcosh | y_true 、 y_pred
tf.keras.applications.ResNet50V2 | input_tensor
tf.nn.moments | x
tf.keras.applications.EfficientNetB4 | input_tensor
tf.image.adjust_jpeg_quality | image
tf.keras.losses.sparse_categorical_crossentropy | y_true 、 y_pred
tf.control_dependencies | control_inputs
tf.image.grayscale_to_rgb | images
tf.image.rgb_to_yiq | images
tf.keras.applications.EfficientNetB3 | input_tensor
tf.ragged.boolean_mask | data、mask
tf.image.random_hue | image
tf.image.adjust_gamma | image
tf.is_tensor | x
tf.keras.initializers.orthogonal | gain
tf.math.polyval | coeffs、x
tf.io.serialize_tensor | tensor
tf.image.stateless_random_saturation | image
tf.one_hot | indices
tf.linalg.diag_part | input、padding_value、
tf.image.adjust_saturation | image
tf.boolean_mask | tensor、mask
tf.transpose | a
tf.image.flip_up_down | image
tf.keras.losses.binary_crossentropy | y_true 、 y_pred
tf.broadcast_to | input
tf.image.stateless_random_crop | value
tf.losses.mean_absolute_percentage_error | y_true 、 y_pred
tf.image.stateless_random_flip_left_right | image
tf.image.random_flip_up_down | image
tf.keras.activations.exponential | x
tf.keras.applications.Xception | input_tensor
tf.identity | input
tf.gather | params
tf.keras.applications.InceptionV3 | input_tensor
tf.keras.layers.Masking | mask_value
tf.losses.kullback_leibler_divergence | y_true 、 y_pred
tf.linalg.band_part | input
tf.keras.losses.cosine_similarity | y_true 、 y_pred
tf.image.random_contrast | image
tf.image.transpose | image
tf.stop_gradient | input
tf.strings.bytes_split | input
tf.random.stateless_parameterized_truncated_normal | means、stddevs、minvals、maxvals
tf.keras.losses.mean_absolute_error | y_true 、 y_pred
tf.image.stateless_random_hue | image
tf.keras.applications.DenseNet169 | input_tensor
tf.keras.losses.categorical_crossentropy | y_true 、 y_pred
tf.nn.embedding_lookup | params
tf.math.reduce_variance | input_tensor
tf.keras.utils.unpack_x_y_sample_weight | data
tf.nn.l2_normalize | x
tf.keras.losses.categorical_hinge | y_true 、 y_pred
tf.keras.applications.EfficientNetB1 | input_tensor
tf.keras.constraints.get | identifier
tf.initializers.orthogonal | gain
tf.divide | x、y
tf.math.top_k | input
tf.keras.losses.kullback_leibler_divergence | y_true 、 y_pred
tf.image.stateless_random_jpeg_quality | image
tf.keras.losses.mean_absolute_percentage_error | y_true 、 y_pred
tf.keras.applications.EfficientNetB7 | input_tensor
tf.clip_by_value | t
tf.type_spec_from_value | value
tf.losses.mean_squared_logarithmic_error | y_true 、 y_pred
tf.tensor_scatter_nd_update | tensor、indices
tf.equal | x、y
tf.image.rgb_to_grayscale | images
tf.image.stateless_random_contrast | image
tf.image.rgb_to_hsv | images
tf.convert_to_tensor | value
tf.losses.sparse_categorical_crossentropy | y_true 、 y_pred
tf.keras.activations.sigmoid | x
tf.slice | input_
tf.image.adjust_hue | image
tf.math.argmax | input
tf.reverse_sequence | input
tf.losses.categorical_crossentropy | y_true 、 y_pred
tf.keras.losses.squared_hinge | y_true 、 y_pred
tf.squeeze | input
tf.math.equal | x、y
tf.math.divide | x、y
tf.unstack | value
tf.keras.applications.MobileNet | input_tensor
tf.keras.applications.ResNet152V2 | input_tensor
tf.keras.activations.softsign | x
tf.keras.applications.NASNetMobile | input_tensor
tf.keras.activations.swish | x
tf.metrics.categorical_accuracy | y_true 、 y_pred
tf.keras.metrics.sparse_categorical_accuracy | y_true 、 y_pred
tf.metrics.sparse_top_k_categorical_accuracy | y_true 、 y_pred
tf.losses.mean_absolute_error | y_true 、 y_pred
tf.losses.binary_crossentropy | y_true 、 y_pred
tf.keras.applications.ResNet50 | input_tensor
tf.image.random_jpeg_quality | image、min_jpeg_quality、max_jpeg_quality
tf.keras.activations.hard_sigmoid | x
tf.image.flip_left_right | image
tf.keras.applications.ResNet101V2 | input_tensor
tf.nn.batch_normalization | x、mean、variance、offset、scale
tf.math.reduce_min | input_tensor
tf.keras.applications.ResNet101 | input_tensor
tf.math.not_equal | x、y
tf.image.rot90 | image
tf.keras.applications.VGG19 | input_tensor
tf.image.stateless_random_flip_up_down | image
tf.keras.applications.MobileNetV3Large | input_tensor
tf.keras.applications.ResNet152 | input_tensor
tf.keras.metrics.categorical_accuracy | y_true 、 y_pred
tf.sparse.cross | inputs
tf.keras.losses.mean_squared_logarithmic_error | y_true 、 y_pred
tf.image.random_flip_left_right | image

</body>

</html>

### Standalone code to reproduce the issue

```shell
Many parameters are not for any type of value, so the allowed types should be clearly marked in the document.
```

### Relevant log output

_No response_</details>

**Comment by tilakrayal** [comment]
@cheyennee,
For some of the API's which were mentioned above, I can see the description was available on the official document. Could you please be more specific where the parameters are missing type descriptions. It will be easy to analyse the issue. Thank you!
